app/assets/javascripts/workflows.js.coffee

```diff
@@ -84,6 +84,15 @@ $(window).focus ->
   else
     $("#script-details-panel").hide()
 
+@show_job_array_request = (show) ->
+  if show
```

Comment:
Intrestingly the other methods are wrong and this is right!

In coffeescript `?` is the "existential operator" and the Ruby equivalent is `nil?`. All the other methods say `if show?` when they should say `if show`. Just interesting. things work, no need to make broad changes.